src/Symfony/Component/HttpFoundation/RedirectResponse.php

```diff
@@ -76,6 +83,10 @@ public function getTargetUrl()
      */
     public function setTargetUrl($url)
     {
+        if (!is_string($url)) {
```

Comment:
100% agree and I made it at first https://github.com/funivan/symfony/commit/79038731461e2d694d4c96be0cd8db108ddae343

But then according to the comments I change this behaviour. 
If cast to string will be better choice, should I change phpdoc comment?